lua/cfc_entity_stubber/main_loader.lua

```diff
@@ -0,0 +1,79 @@
+cfcEntityStubber = { }
+cfcEntityStubber.stubQueue = { }
+cfcEntityStubber.stubbers = { }
+cfcEntityStubber.oldWeaponStats = { }
+cfcEntityStubber.newWeaponStats = { }
```

Comment:
these arent used yet. do you intend to implement that functionality in a later pr?

Reply:
oldWeaponStats is used in cfcEntityStubber.getWeapon
i'll remove newWeaponstats